Solutions/Corelight/Data Connectors/Connector_Exporter_Corelight.json

```diff
@@ -0,0 +1,550 @@
+{
+  "id": "Corelight",
+  "title": "Corelight",
+  "publisher": "Corelight",
+  "descriptionMarkdown": "The [Corelight](https://corelight.com/) data connector enables incident responders and threat hunters who use Microsoft Sentinel to work faster and more effectively. The data connector enables ingestion of events from [Zeek](https://zeek.org/) and [Suricata](https://suricata-ids.org/) via Corelight Sensors into Microsoft Sentinel.",
+  "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**Corelight**](https://aka.ms/sentinel-Corelight-parser) which is deployed with the Microsoft Sentinel Solution.",
+  "graphQueries": [
+    {
+      "metricName": "Total data received",
+      "legend": "Corelight_CL",
+      "baseQuery": "Corelight"
+    }
+  ],
+  "sampleQueries": [
+    {
+      "description": "Top 10 Clients (Source IP)",
+      "query": "Corelight\n\t| summarize count() by id_orig_h\n\t| top 10 by count_"
+    }
+  ],
+  "dataTypes": [
+    {
+      "name": "Corelight_v2_UNCATEGORIZED_CL",
```

Comment:
Hello @ekoyle, You have mentioned a parser Corelight_v2_UNCATEGORIZED_CL in the connector.
But is not located in the solution. Try adding the parser in the solution.

Reply:
I have removed this reference as it is not referenced anywhere else in our integration.  Let me know if I need to add anything elsewhere.